fcli-core/fcli-aviator-common/src/main/java/com/fortify/cli/aviator/core/IssueAuditor.java

```diff
@@ -134,31 +137,44 @@ public void performAudit(Map<String, AuditResponse> auditResponses, String token
         LOG.info("Built {} user prompts from vulnerabilities", userPrompts.size());
         ConcurrentLinkedDeque<UserPrompt> filteredUserPrompts = getIssuesToAudit();
         logger.progress("Filtered issues count: %d", filteredUserPrompts.size());
-        try (AviatorGrpcClient client = createClientFromUrl(url)) {
-            CompletableFuture<Map<String, AuditResponse>> future = client.processBatchRequests(filteredUserPrompts, projectName, token);
-            try {
-                Map<String, AuditResponse> responses = future.get(500, TimeUnit.MINUTES);
-                responses.forEach((requestId, response) -> {
-                    auditResponses.put(response.getIssueId(), response);
-                });
-            } catch (ExecutionException e) {
-                LOG.error("Error executing requests: {} ", e.getCause());
-            } catch (TimeoutException e) {
-                LOG.error("Error executing requests:timeout {}", String.valueOf(e.getCause()));
-            } catch (InterruptedException e) {
-                Thread.currentThread().interrupt();
-                LOG.error("Error executing requests:interrupted {}", e.getCause());
+
+        if (filteredUserPrompts.isEmpty()) {
+            logger.progress("Audit skipped - no issues to process");
+            logger.writeInfo("Audit skipped - no issues to process");
+        } else {
+            try (AviatorGrpcClient client = AviatorGrpcClientHelper.createClient(url, logger)) {
+                CompletableFuture<Map<String, AuditResponse>> future = client.processBatchRequests(filteredUserPrompts, projectName, token);
+                try {
```

Comment:
Why do you have a nested `try` block here? You already declare a `try` block in the previous line, and don't access the `client` variable in any of the `catch` blocks, so no need for a nested `try` block I think.